nano/version.py

```diff
@@ -1 +0,0 @@
-__version__ = '2.0.1'
```

Comment:
It's not removed in my local stuff. I don't understand what's up, I think I need to start using another editor because I have no recollection of this or the above change :(